src/components/DashboardResultIndicatorsSection.vue

```diff
@@ -176,6 +183,27 @@ export default {
   computed: {
     ...mapState(['kpis', 'subKpis', 'theme']),
     ...mapGetters(['hasEditRights']),
+    periodTrend(){
+      const sortedProgress = this.progressCollection.sort((a, b) => (a.timestamp > b.timestamp ? 1 : -1));
+      const firstProgressRecord = sortedProgress.slice(0, 1)[0]?.value;
```

Comment:
Trenger ikke å `slice` for å hente ut første, bare [0].

Reply:
Rettet